tests/ui/test_ui_tools.py

```diff
@@ -17,12 +17,11 @@
     StreamButton, TopButton, TopicButton, UserButton,
 )
 from zulipterminal.ui_tools.views import (
-    AboutView, EditModeView, HelpView, LeftColumnView, MessageView,
-    MiddleColumnView, ModListWalker, MsgInfoView, PopUpConfirmationView,
-    PopUpView, RightColumnView, StreamInfoView, StreamsView,
-    StreamsViewDivider, TopicsView, UsersView,
+    EditModeView, HelpView, LeftColumnView, MessageView, MiddleColumnView,
```

Comment:
minor: I know this makes the file lengthier, but we may want to move towards having one import per line. That way we reduce the diffs in rearranging to adjust for line length limit.